publishable/config/voyager.php

```diff
@@ -1,6 +1,46 @@
 <?php
 
 return [
+
+    /*
+    |--------------------------------------------------------------------------
+    | Factory config
+    |--------------------------------------------------------------------------
+    |
+    | Here you can specify what classes should be returned when using e.g Voyager::model('Category')
+    |
+    */
+
+    'factory' => [
+        'models' => [
```

Comment:
Similar comment to the policies config below this.  This was originally done to encourage use of Voyager's as a UI to manage your app, however in this case, there is no UI for the models.  I'm okay with this change, but I think we should at least deprecate the `Voyager::useModel()` method in favor of this config-based approach.

Reply:
I must admit I'm new to the voyager environment, the reason behind me wanting to move these to config was because I'm working on a 'multi-site' sort of application where admins can only see part of the collections, but still have read, edit and delete privileges.

Voyager worked really well except for some edge cases, like these hard-coded values within the class. Thats why I thought it good to extract them to config.

I can solve this by overriding Voyager.php in my own app and override the facade as well, if you don't think this is suited for the overall purpose of voyager.

Reply:
I don't think you need to override the whole class just to change some of these values.  Voyager provides a `useModel()` method on the facade, which lets you override these "hard-coded" model classes.  Policies can be overridden by (re)defining a Policy in your app's AuthServiceProvider, so long as that provider runs after VoyagerServiceProvider (Laravel 5.5's autodiscovery mechanism might make this more difficult, but it's possible)

Reply:
Oh! i completely missed that setting! How do you want me to go forward with this PR then? remove the config or deprecate useModel?